README.md

```diff
@@ -0,0 +1,144 @@
+<table><tr>
+<td><a href="https://gcr.io"><img src="https://avatars2.githubusercontent.com/u/1342004?s=200&v=4" height="120"/></a></td>
+<td><a href="https://gcr.io"><img src="https://avatars2.githubusercontent.com/u/21046548?s=400&v=4" height="120"/></a></td>
+<td><a href="https://bazel.build"><img src="https://bazel.build/images/bazel-icon.svg" height="120"/></a></td>
+</tr></table>
```

Comment:
This isn't doing anything.  Can we add a Travis badge?  @dekkagaijin Are you going to merge this into the mono-repo?

Reply:
Not doing anything? As in, they're static assets?

> Can we add a Travis badge?
inconceivable O.o

Reply:
interesting....  earlier those images weren't even loading for me.  We should add a Travis badge, but IDK what caused my original problem.